model/src/main/java/keywhiz/model/TinyIntConverter.java

```diff
@@ -16,9 +16,11 @@
 
 package keywhiz.model;
 
+import edu.umd.cs.findbugs.annotations.SuppressFBWarnings;
 import org.jooq.Converter;
 
 public class TinyIntConverter implements Converter<Byte, Boolean> {
+  @SuppressFBWarnings(value = "NP_BOOLEAN_RETURN_NULL", justification = "null is a valid DB value")
   @Override public Boolean from(Byte aByte) {
```

Comment:
We could write this as `Converter<Byte, Optional<Boolean>>` and force callers to handle the `Optional.empty` case?